File: python_draft/src/cpp/type2/process_data_renamed.cpp
#include <vector>
std::vector<int> processDataV1(const std::vector<int>& dataValues){
    std::vector<int> output;
    for(int value : dataValues){
        if(value % 2 == 0) output.push_back(value * 2);
    }
    return output;
}

File: python_draft/src/java/type2/ProcessDataRenamed.java
public class ProcessDataRenamed {
public static java.util.List<Integer> processDataV1(java.util.List<Integer> dataValues) {
    java.util.List<Integer> output = new java.util.ArrayList<>();
    for (int value : dataValues) {
        if (value % 2 == 0) {
            output.add(value * 2);
        }
    }
    return output;
}

}
